nuopc_dart.F90

```diff
@@ -24,7 +24,7 @@ module dart_comp_nuopc
     use NUOPC            , only : NUOPC_CompAttributeGet, NUOPC_Advertise
     use NUOPC            , only : NUOPC_CompFilterPhaseMap
     use NUOPC, only : NUOPC_Write ! HK what is the write for?
-    use NUOPC, only : NUOPC_SetAttribute, NUOPC_CompAttributeSet, NUOPC_Realize
+    use NUOPC, only : NUOPC_SetAttribute, NUOPC_CompAttributeSet, NUOPC_Realize, NUOPC_GetAttribute
 
     !HK three of these rename on import did not seem to work, using the model_ version for now
     use NUOPC_Model, model_routine_SS        => SetServices 
```

Comment:
I have no idea why it's not working. I have seen all the codes where people rename this like this!